app/search/home.tsx

```diff
@@ -60,7 +61,7 @@ export default function SearchHome() {
   const exportAction = useExport()
   const columns = useColumns()
   const pluginButtons = usePluginToolbarItems("search")
-
+  const poolId = useLakeId()
```

Comment:
we should probably rename this hook to `usePoolId()`

Reply:
Lots of places need to be renamed. I'll make a ticket.

Reply:
https://github.com/brimdata/brim/issues/1922